public/app/features/plugins/admin/components/Changelog.tsx

```diff
@@ -0,0 +1,49 @@
+import { css, cx } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+import { CatalogPlugin } from '../types';
+
+interface Props {
+  plugin: CatalogPlugin;
+}
+
+export const Changelog = ({ plugin }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div
+      dangerouslySetInnerHTML={{ __html: plugin.details?.changelog ?? 'No changelog was found' }}
+      className={cx(styles.changelog, styles.container)}
+    ></div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  container: css`
+    padding: ${theme.spacing(2, 4, 3)};
+  `,
+  changelog: css({
+    h1: {
```

Comment:
nit: I'm guessing this is due to most changelogs having `# Changelog`? Might be worth using `first-of-type` here too so we don't hide info in changelogs with different formats?